user/views.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from rest_framework import generics
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.settings import api_settings
+
+from user.serializers import UserSerializer
+
+
+class CreateUserView(generics.CreateAPIView):
+    serializer_class = UserSerializer
+    permission_classes = ()
```

Comment:
?

Reply:
I mean why is it blank 😁

Reply:
fixed and changed to AllowAny, it's needed because I used the custom permission class as default in settings, so we need to allow anyone to register